tests/php/Dev/CSVParserTest.php

```diff
@@ -24,7 +25,10 @@ protected function setUp(): void
     public function testParsingWithHeaders()
     {
         /* By default, a CSV file will be interpreted as having headers */
-        $csv = new CSVParser($this->csvPath . 'PlayersWithHeader.csv');
+        $filepath = $this->csvPath;
```

Comment:
Instead of wrapping calls here, instead change setUp() to the following:

```php
    protected function setUp(): void
    {
        parent::setUp();
        $this->csvPath = __DIR__ . '/CsvBulkLoaderTest/csv/';
        if (Deprecation::isEnabled()) {
            $this->markTestSkipped('Test calls deprecated code');
        }
    }
```

Reply:
Done